### PR TITLE
Shell+LibLine: Some nice fixes

### PR DESCRIPTION
**Pull request by alimpfard**

### CODE REVIEW

File: Userland/Libraries/LibLine/Editor.cpp [hunk 1]
```diff
@@ -2106,7 +2112,7 @@ Result<Vector<size_t, 2>, Editor::Error> Editor::vt_dsr()
     if (m_input_error.has_value())
         return m_input_error.value();
 
-    fputs("\033[6n", stderr);
+    fputs("\033[6n\n", stderr);
```
Comment (nico) [inline Userland/Libraries/LibLine/Editor.cpp:2115]: Is this in here intentionally? Looks somewhat unrelated to the commit message from a distance.

**Review by nico — APPROVED** [review]
nice

File: Userland/Libraries/LibLine/Editor.cpp (hunk 1 above)
Comment (alimpfard) [reply Userland/Libraries/LibLine/Editor.cpp:2115]: It's intentional, should be a separate commit though, will fix.
